### ISSUE TITLE
Player's AP is reset to max when exiting inventory after using up all APs

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#314]
```
What steps will reproduce the problem?
1. Engage in combat (with 10AP)
2. Go into inventory
3. Equip some item (using 5AP)
4. Unequip that item (using 5AP)
5. Now you should have 0AP
6. Return to combat
7. Notice how the monsters do not attack, but instead you have full AP.

Instead, when returning to combat, you should have 0AP, and would have to press 
"end turn".

Confirmed in v0.6.11.

```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 24 Jul 2012 at 6:13

**Comment by GoogleCodeExporter** [comment]
```
Fixed in e3d49008872c.
```

Original comment by `oskar.wi...@gmail.com` on 24 Jul 2012 at 6:14
- Changed state: **Ready**

**Comment by GoogleCodeExporter** [comment]
```
Thanks to Drizzt on forums for finding it.
```

Original comment by `oskar.wi...@gmail.com` on 24 Jul 2012 at 6:19

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.12.
```

Original comment by `oskar.wi...@gmail.com` on 28 Oct 2012 at 7:43
- Changed state: **Fixed**
- Added labels: **Milestone-0.6.12**
- Removed labels: **Milestone-0.6.11**
